### ISSUE TITLE
Extension Causes Instant Ban on Ome.tv

### ISSUE THREAD

**Issue by truongakevin** [videochat-extension/videochat-extension#57]
When using this Chrome extension for Ome.tv, I am immediately banned upon opening the website. A banner appears with the message: “You’re banned. Reason: Using third-party software to alter the service functionality.”

I cleared my cookies and created a new account but I got banned instantly again. This seems to occur the moment I load the site with the extension enabled.

Is there a way to prevent the extension from triggering an automatic ban? Is this happening to anyone else?

System Details:
	•	Browser: Chrome Version 131.0.6778.109 (Official Build) (arm64)
	•	OS: MacOS Sonoma

Any advice or guidance would be greatly appreciated!

**Comment by qrluke** [comment]
Yes, this platform silently changed its policy and began banning some users. Videochat Extension has been rebuilt to completely distrust the website, and this repository is going to be archived.

We received only 30 reports from a user base of 200,000, mostly from individuals evading bans, so it is unlikely that this was a widespread problem. We also disabled the ome.tv support after receiving such reports. However, our investigation concluded that peaceful coexistence is no longer an option for this project, so the old extension cannot be saved.
